sdk/resourcemanager/azure-resourcemanager-network/CHANGELOG.md

```diff
@@ -5,7 +5,7 @@
 ### Features Added
 
 - The class `RoutingConfigurationNfv` renamed to `RoutingConfiguration`.
-- Remove classes `PropagatedRouteTableNfv` and `RoutingConfigurationNfvSubResource`.
+- The classes `PropagatedRouteTableNfv` and `RoutingConfigurationNfvSubResource` removed.
```

Comment:
These are breaking changes. Please add back breaking changes section and put them under breaking changes section.

Reply:
Fixed in the new version.